django_celery_fulldbresult/models.py

```diff
@@ -1,12 +1,23 @@
 from __future__ import absolute_import, unicode_literals
+
+import json
+
+from django.conf import settings
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
+try:
+    from django.utils.encoding import force_text
+except ImportError:
+    from django.utils.encoding import force_unicode as force_text  # noqa
```

Comment:
django celery has similar comments if I remember correctly. This comment is also supported by pep8, which is kind of a standard.